easybuild/easyconfigs/p/pod5-file-format/pod5-file-format-0.1.8-foss-2022a.eb

```
@@ -0,0 +1,86 @@
+easyblock = 'CMakeMake'
+
+name = 'pod5-file-format'
+version = '0.1.8'
+
+homepage = 'https://github.com/nanoporetech/pod5-file-format'
+description = """POD5 is a file format for storing nanopore dna data in an easily accessible way.
+ The format is able to be written in a streaming manner which allows a sequencing
+ instrument to directly write the format."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+source_urls = ['https://github.com/nanoporetech/%(name)s/archive/']
+sources = [{'download_filename': '%(version)s.tar.gz', 'filename': SOURCE_TAR_GZ}]
+patches = ['pod5-file-format-0.1.8_dep_fix.patch']
+checksums = [
+    {'pod5-file-format-0.1.8.tar.gz': '262b85a44b2e5e93be55bfc2d55a49c07a012b04b59aa24e1d59fd981ac58b5c'},
+    {'pod5-file-format-0.1.8_dep_fix.patch': 'd24fab7477d735ba7bb4a286c7a64d6138bf6a82a5a021b257eeeb5dfa06d82e'},
+]
+
+builddependencies = [
+    ('binutils', '2.38'),
+    ('CMake', '3.23.1'),
+]
```

Rather than installing `pkgconfig` as an extension and including `pkg-config` as a runtime dependency, let's just use the existing `pkgconfig-1.5.5-GCCcore-11.3.0-python.eb` as a build dependency instead:

```suggestion
builddependencies = [
    ('binutils', '2.38'),
    ('CMake', '3.23.1'),
    ('pkgconfig', '1.5.5', '-python'),
]